src/components/TextInput/index.js

```diff
@@ -7,65 +7,66 @@ import * as baseTextInputPropTypes from './baseTextInputPropTypes';
 import DomUtils from '../../libs/DomUtils';
 import Visibility from '../../libs/Visibility';
 
-class TextInput extends React.Component {
-    componentDidMount() {
-        if (this.props.disableKeyboard) {
-            this.textInput.setAttribute('inputmode', 'none');
+function TextInput(props) {
```

Comment:
As per [the style guide](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring), it is suggested to destructure the props.

> Always use destructuring to get prop values. Destructuring is necessary to assign default values to props.

Reply:
But we do pass props as it is to BaseTextInput. So, I guess it is okay to use props as parameter.